src/pudl_archiver/zenodo/api_client.py

```diff
@@ -232,10 +232,6 @@ async def create_deposition(self, data_source_id: str):
             # Ignore content type
             return Deposition(**await self._check_resp(response, content_type=None))
 
-    def changed(self) -> bool:
```

Comment:
With the "clear pending changes after they're applied" update below, this became "are there outstanding changes to apply to the deposition" and not "were there ever any changes."